licenses/tests/test_models.py

```diff
@@ -162,10 +162,61 @@ def test_branch_name(self):
         self.assertEqual("other-35-de-xyz", legalcode.branch_name())
 
 
-# Many of these tests mostly are based on whether the metadata import worked right, and
-# we're not importing metadata for the time being.
 class LicenseModelTest(TestCase):
-    # fixtures = ["licenses.json"]
+    def test_get_metadata(self):
+        license = LicenseFactory(
+            **{
+                "license_code": "by-nc",
+                "version": "3.0",
+                "title_english": "The Title",
+                "jurisdiction_code": "xyz",
+                "permits_derivative_works": False,
+                "permits_reproduction": False,
+                "permits_distribution": True,
+                "permits_sharing": True,
+                "requires_share_alike": True,
+                "requires_notice": True,
+                "requires_attribution": True,
+                "requires_source_code": True,
+                "prohibits_commercial_use": True,
+                "prohibits_high_income_nation_use": False,
+            }
+        )
+
+        LegalCodeFactory(license=license, language_code="pt")
+        LegalCodeFactory(license=license, language_code="en")
+
+        data = license.get_metadata()
+        expected_data = {
+            "license_code": "by-nc",
+            "version": "3.0",
+            "title_english": "The Title",
+            "jurisdiction": "xyz",
+            "permits_derivative_works": False,
+            "permits_reproduction": False,
+            "permits_distribution": True,
+            "permits_sharing": True,
+            "requires_share_alike": True,
+            "requires_notice": True,
+            "requires_attribution": True,
+            "requires_source_code": True,
+            "prohibits_commercial_use": True,
+            "prohibits_high_income_nation_use": False,
+            "translations": {
+                "en": {
```

Comment:
This looks really good! Should we include percent translate for each dictionary in `translations`?

Reply:
Should we also include the `cc-licenses-data` branch name in this section

Reply:
I feel like the percent translated and the branch name are more artifacts of the translation process than characteristics of the licenses themselves which I think of as being the metadata, but I'll go by whatever @TimidRobot thinks.